src/sst/elements/merlin/topology/pymerlin-topo-any.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python
+# TODO: copyright goes here?
+
+"""
+SST Merlin Any Topology Module
+
+This module provides support for arbitrary network topologies using NetworkX graphs.
+It supports source routing mode with routing table calculation.
+
+For using source routing with this topology:
+- Import graph: topo.import_graph(graph_input)
+- Calculate routing table: routing_table = topo.calculate_routing_table()
+- Pass to SourceRoutingPlugin via EndpointNIC.addPlugin()
+
+See pymerlin-interface.py for SourceRoutingPlugin usage and routing table serialization.
+"""
+
+import sst
+from sst.merlin.base import *
+import os
+from collections import defaultdict
+from typing import Callable, Optional
+
+try:
```

Comment:
This looks like the code segment that is failing when networks isn't installed.  It looks like this code is executing when the module is loaded, so the ImportError below ends up failing in the module loading.  So, this causes every simulation that imports sst.merlin to fail, not just those that use topoany.  Probably need to set a variable if the modules are missing and fail when trying to use the topology instead of at module load time.